File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i = len(s) - 1
        count = 0

        while i >= 0:
            if s[i] == " ":
                if count == 0:
                    i -= 1
                    continue
                else:
                    break
            else:
                i -= 1
                count += 1
        return count

File: 0058-length-of-last-word/0058-length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function (s) {
    let i = s.length - 1;
    let count = 0;

    while (i >= 0) {
        if (s[i] === " ") {
            if (count === 0) {
                i--;
                continue;
            } else {
                break;
            }
        } else {
            i--;
            count++;
        }
    }

    return count;
};